cms/migrations/0022_merge_20220805_1914.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 3.2.14 on 2022-08-05 19:14
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cms", "0020_add_productpage_and_base_tables"),
+        ("cms", "0021_flexiblepricingrequestform_selected_program"),
+    ]
+
+    operations = []
```

Comment:
It looks like this migration is empty?

Reply:
It's a merge migration; it doesn't have any database operations in it. (There was a migration 20 and 21 in `cms` in a different branch, so this provides a spot for future migrations to base their dependencies on.)

Reply:
I see, usually I would undo my migrations, rebase my branch and then create new migrations. I think it works both ways, but I am wondering if we have a preferred way of doing this.